sched/sched/sched_waitid.c

```diff
@@ -167,9 +167,11 @@ int nx_waitid(int idtype, id_t id, FAR siginfo_t *info, int options)
        */
 
       ctcb = nxsched_get_tcb((pid_t)id);
-      if (ctcb != NULL)
+      if (ctcb && ctcb->group)
```

Comment:
```suggestion
      if (ctcb != NULL && ctcb->group != NULL)
```

Reply:
Please use the log form in this PR.

Reply:
ctcb is a pointer, why must if (ctcb != NULL) ?
Why not use if (ctcb) ?
If there is a coding guideline say this, show me.